public/web.js

```diff
@@ -0,0 +1,557 @@
+var lib = (() => {
```

Comment:
was this file generated automatically?

Reply:
yup, added a super mini build system called `esbuild` to bundle and transpile all the class-based stuff to more browser-compatible format. You can see the `package.json` has a few scripts now that build and run the server. One of the next tickets/issues might be to integrate `express` so we can do better routing

Reply:
okay, that's what I figured, but had some mild panic when I looked at this file and went WHAT is THIS. 
what's the reason to avoid React? am I unnecessarily biased for asking that question?

Reply:
and cool, noted about the scripts. guessing `test` doesn't do anything yet, haha
maybe worth adding some notes about running it to the README?

Reply:
yea fair point bout the README.

correct, `test` does nothing at the moment.

React - no particular reason other than I know how to program in React - this was an opp to use WebComponents, which I have been aware of but never got the chance to actually use. It does hinder us using React Native tho to make it a possibly mobile native implementation. But that could be interesting as a separate repo. But this is a fine application for which to use React, esp if we ever care to start implementing features like user profiles & game history or other "stateful" data needs